packages/common/src/services.ts

```diff
@@ -820,6 +794,38 @@ export class CondaPackage implements Conda.IPackageManager {
     return Promise.resolve(CondaPackage._availablePackages);
   }
 
+  /**
+   * Get the list of installed packages
+   *
+   * @param {string} environment The environment in which the list of packages is
+   * installed; if none is provided, defaults to the current environment
+   * @return {Promise<Array<Conda.IPackage>>} Array of installed packages.
```

Comment:
The thing I don't like about this is that it suggests to consumer that this method returns an array of objects (Conda.IPackage) that are ready for the Gator UI. But that's misleading. The packages returned by this method will meet the type definition for IPackage, but they will have empty strings for some important fields, like package summary. 

The only two ways out of this that I see are:
- change the server side code (to return IPackage instead of IRawPackage)
- change the function signature

Changing the server-side code feels out of scope for this PR. 

Changing the function signature is also problematic because then I'd have to change it on the Conda Store model, or I'd have to create two different interfaces, one for this class and one for the other (because I don't believe it makes sense to overload the method definition because there's no reason that the Conda Store model should return an array of `RESTAPI.IRawPackage`, and if you don't overload the method in both classes, then I believe you have to have two different interfaces).